packages/react-core/src/layouts/Gallery/__snapshots__/Gallery.test.js.snap

```diff
@@ -0,0 +1,37 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`gutter lg 1`] = `
+.pf-l-gallery.pf-m-gutter {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(250px, 1fr));
+  grid-gap: 1.5rem;
+}
+
+<div
```

Comment:
are these snapshots supposed to be rendering different classes / css for the gutter size? they all seem to be rendering the same css/markup...

Reply:
Not yet. Gutters will change soon, though. Right now there is only one size, but more will come soon. This will let is know when that happens.

Reply:
do you still want to update this or is it OK to merge as is? I don't see it as a blocker and it seems like other PRs depend on this.

Reply:
Yeah.  The gutter stuff will come at some point, but I am unsure when.  Should not be a blocker.  This PR should be good to go now that I rebased.

Reply:
ok - i definitely won't be able to test this downstream (not a consumer at this point), so trust your judgement there...approving.